### ISSUE TITLE
A question on grad_times_image.py

### ISSUE THREAD

**Issue by medinabaitemirova** [utkuozbulak/pytorch-cnn-visualizations#84]
Hi!

In grad_times_image.py, line 23:

`grad_times_image = vanilla_grads[0] * prep_img.detach().numpy()[0]
`

Shouldn't vanilla_grads[0] be vanilla_grads? Otherwise, it is only using the first channel of the gradient. 

Thank you!

**Comment by utkuozbulak** [comment]
Hey,

That's correct. It was the is batch channel but I refactored a couple of things after pushing that code which probably messed things up.

**Comment by utkuozbulak** [comment]
Updated with https://github.com/utkuozbulak/pytorch-cnn-visualizations/commit/7dd09a0fc04abcd2a4a3ddb47b8ea230732b8a94 , thanks for heads up.
